x-pack/solutions/security/plugins/security_solution/public/helpers_access.ts

```diff
@@ -11,7 +11,6 @@ export function hasAccessToSecuritySolution(capabilities: Capabilities): boolean
   return Boolean(
     // Using `siemV2`
     capabilities[SECURITY_FEATURE_ID]?.show ||
-      capabilities.securitySolutionCasesV2?.read_cases ||
```

Comment:
Can't we just change `securitySolutionCasesV2` by `securitySolutionCasesV3`, instead of doing the cases check outside?

Reply:
We can't without changing the behavior. The redirect would get into the `/get_started` page instead of `/cases` here: https://github.com/elastic/kibana/blob/1b167d9dc23a9e0e8e47992a37563ca89ccf3c7d/x-pack/solutions/security/plugins/security_solution/public/app/app_routes.tsx#L36

Reply:
@tomsonpl Oh, good point. 
Then would you mind creating different functions, with a clear naming, because this is a bit confusing right now. We could have:

- hasSecurityAccess -> `security || attack-discovery` (used in app_routes)
- hasCasesAccess ->  `cases` (used in app_routes)
- isSecuritySolutionAccessible ->  `hasSecurityAccess() || hasCasesAccess()` (used in the plugin.tsx)

And consider using `CASES_FEATURE_ID` instead of hardcoding the version.

Question: Should we consider letting users access Security if they only have the `assistant` feature? Maybe it should be part of `hasSecurityAccess`.